distro/license-book/src/main/mustache/license-book.tpl

```diff
@@ -0,0 +1,14 @@
+# License Book for Operaton {{version}}
+
+{{>toc}}
+
+{{>preamble}}
+
+{{>license-summary}}
+
+{{> license}}
```

Comment:
Do we really need to include each license text here? if yes, could we place it at the very end of this file?